### PR TITLE
DOCS-2735 Style Review Snapshots Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
Reviewed and edited article for style changes, grammar voice and tense cases. Added several snippets for text common to CORE and SCALE.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/CORE/Storage/Snapshots.md [hunk 1]
```diff
@@ -5,39 +5,26 @@ weight: 20
...
-{{< /expand >}}
+{{< include file="/_includes/SnapshotsIntroSnippet.md" type="page" >}}
 
 Taking snapshots requires the system have all [pools]({{< relref "/CORE/Storage/Pools/_index.md" >}}), [datasets]({{< relref "" >}}), and [zvols]({{< relref "" >}}) already configured.
 
 ## Creating a Single Snapshot
 
 {{< hint ok >}}
-To save time and regularly create fresh snapshots, consider making a [Periodic Snapshot Task]({{< relref "PeriodicSnapshotTasks.md" >}}) instead.
+Consider making a [Periodic Snapshot Task]({{< relref "/SCALE/DataProtection/_index.md" >}}) to save time and create regular, fresh snapshots.
```
Comment (Mrt134) [inline content/CORE/Storage/Snapshots.md:15]: This reference needs to point to the CORE content location, not SCALE.

**Review by Mrt134 — CHANGES_REQUESTED** [review]
One reference to adjust, everything else looks good!

**Comment by micjohnson777** [comment]
Oops... did that happen because I copy/pasted an incorrect link? And did
you merge this but not publish so I can just open that PR up, make the
change and recommit it after I fix? Just checking to make sure before I
take action.

File: content/CORE/Storage/Snapshots.md (hunk 1 above)
Comment (micjohnson777) [reply content/CORE/Storage/Snapshots.md]: issue fixed, PR is currently rebuilding.

**Comment by Mrt134** [comment]
Exactly - I haven't merged this PR into the master branch yet, but put the comment in place so that you can fix in your branch and update before we merge.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**